src/frontend/src/optimizer/optimizer_context.rs

```diff
@@ -160,6 +165,13 @@ impl OptimizerContext {
         optimizer_trace.push("\n".to_string());
     }
 
+    pub fn warn(&self, str: impl Into<String>) {
```

Comment:
Any usage currently?

Reply:
not yet

Reply:
[not yet](https://github.com/risingwavelabs/risingwave/issues/9582)